design-reviews/recipes/milestone-epic-design-review-recipe.md

```diff
@@ -0,0 +1,79 @@
+* This is *not* a strict template that must be followed and teams should not be bogged down with polished "design presentations".
+* Think of the recipe below as a "menu of options" for potential questions to think through in designing this epic. Not all sections are required for every epic. Focus on sections and questions that are most relevant for making the decision and rationalizing the trade offs.
+* Milestone/epic design is considered high-level design but is usually more detailed than the design included in the Game Plan, but will likely re-use some of the technologies, non-functional requirements, and constraints mentioned in the Game Plane.
+* As the team learned more about the project and further refined the scope of the epic, they may specifically call out notable changes to the overall approach and, in particular, highlight any unique deployment, security, private, scalability, etc. characteristics of this milestone.
+
+***
+
+# [Design Title]
+
+* Milestone / Epic: [Name](http://link-to-work-item)
+* Project / Engagement: [Project Engagement]
+* Authors: [Authors]
+
+## Overview / Problem Statement
+
+> Describe the milestone/epic with a high-level summary and a problem statement. Consider including or linking to any additional background (e.g. Game Plan or Checkpoint docs) if it is useful for historical context.
+
+## Goals / In-Scope
+
+> List bullet points of goals that the milestone/epic will help achieve. Include acceptable criteria required to meet the [Definition of Done](../../team-agreements/definition-of-done/).
+
+## Non-goals / Out-of-Scope
+
+> List bullet points of non-goals for this milestone/epic to clarify the work that is beyond the scope of this milestone/epic.
+
+## Proposed Design / Suggested Approach
+
+* Describe the proposed design / suggested approach for this milestone/epic.
+* List some of the alternative approaches that were considered.
+* Include relevant diagrams (e.g. architecture, sequence, component, deployment, etc.) and code/pseudo-code snippets to make it easier to talk through the approaches.
+* Whenever possible, for each of the approaches, include the brief key **Pros and Cons** used to help rationalize the decision. For example:
+
+| Pros                             | Cons                                  |
+| -------------------------------- | ------------------------------------- |
+| Simple to implement              | Creates secondary identity system     |
+| Repeatable pattern/code artifact | Deployment requires admin credentials |
+|                                  |                                       |
+
+## Technology
+
+> Describe the languages(s) and platform(s) that comprise the stack. This may include anything that is needed to understand the overall solution: OS, web server, presentation layer, persistence layer, caching, eventing, etc.
+
+## Operationalization

```

Comment:
Can we condense/consolidate Operationalization, Performance, etc to Non-Functional Requirements? For example, latency tolerance of 50ms, 3 9's availability, RPO/RTO objectives, Cost contraints (x cost per transaction). NFR's is fairly industry standard term for representing these more technical and operational concerns.

Reply:
Changed the Perf section to NFR and moved it higher. Kept Operationalization section separate for now since ops maybe slightly different than requirements and list of questions to potentially think through in a combined section may look to long.